dts/bindings/led/ti,lp5569.yaml

```diff
@@ -3,3 +3,16 @@ description: TI LP5569 LED
 compatible: "ti,lp5569"
 
 include: i2c-device.yaml
+
+properties:
+  charge-pump-mode:
```

Comment:
Also it is worth mentioning this new property in the 4.0 release notes.

Reply:
Added